### PR TITLE
[studio-ui] Path Navigator update to restore stored state on the component instead of epic

### PR DESCRIPTION
**Pull request by aachavarria**

### CODE REVIEW

File: ui/app/src/state/epics/pathNavigator.ts [hunk 1]
```diff
@@ -34,37 +34,38 @@ import {
...
+              id,
+              path: state.pathNavigator[id].currentPath,
+              offset: state.pathNavigator[id].offset,
+              keyword: state.pathNavigator[id].keyword,
+              excludes: excludes,
+              limit: state.pathNavigator[id].limit
+            })
+          );
+        }
+      )
```
Comment (rart) [inline ui/app/src/state/epics/pathNavigator.ts]:
```suggestion
      map(
        ([
          {
            payload: { id, excludes }
          },
          state
        ]) => 
            pathNavigatorFetchParentItems({
              id,
              path: state.pathNavigator[id].currentPath,
              offset: state.pathNavigator[id].offset,
              keyword: state.pathNavigator[id].keyword,
              excludes,
              limit: state.pathNavigator[id].limit
            })
      )
```

File: ui/app/src/state/epics/pathNavigator.ts [hunk 2]
```diff
@@ -19,7 +19,7 @@ import { ignoreElements, map, mergeMap, switchMap, tap, withLatestFrom } from 'r
 import { catchAjaxError } from '../../utils/ajax';
 import { fetchChildrenByPath, fetchItemsByPath, fetchItemWithChildrenByPath } from '../../services/content';
 import { getIndividualPaths, getRootPath } from '../../utils/path';
-import { forkJoin } from 'rxjs';
+import { forkJoin, of } from 'rxjs';
```
Comment (rart) [inline ui/app/src/state/epics/pathNavigator.ts]:
```suggestion
import { forkJoin } from 'rxjs';
```

**Review by rart — CHANGES_REQUESTED** [review]
Comments above

**Review by rart — APPROVED** [review]

**Comment by sumerjabri** [comment]
@aachavarria conflict, please fix.

**Review by rart — APPROVED** [review]

**Pull request merged**